gorequest.go

```diff
@@ -74,6 +74,7 @@ type SuperAgent struct {
 		Attempt         int
 		Enable          bool
 	}
+	DoNotClearSuperAgent bool
```

Comment:
It would be awesome if you could add a function to set the flag? 
(Please see SetDebug or SetCurl)

Also, please comment over function and add documentation to README.md to explain about this new feature.